molecule/docker/tests/test_playbook.py

```diff
@@ -0,0 +1,4 @@
+def test_kdump_running_and_enabled(host):
```

Comment:
These tests mainly verify that ansible's `service` module worked. Is there something special about the chronyd service that makes you think it might stop or become disabled after the playbook starts and enables it? If so, please add a comment indicating this. If not, I think these tests are probably redundant since the converge playbook should fail if the service fails to start.

Reply:
Personally I've many times had the `service` command or the `service` module start up a service, particularly a network client service like this, that later (5-10 seconds time frame) just shut itself down because it couldn't do something it wanted to do, like reach a remote host, etc.

That might be worth testing here, in case our list of default hosts is later invalidated and chrony fails because of it?

Reply:
Like I said, if there's something special about the chrony service that means we can't trust ansible's `service` module to be "good enough", then it certainly is worth testing but we should add a comment explaining why this isn't a redundant test.

Reply:
It looks like chrony doesn't have anything wacky that would cause it to fail after configuration, and if the configuration file generated has a service breaking error, the playbook fails. I'll go ahead and remove the testinfra tests